modules/gql.yml

```diff
@@ -0,0 +1,17 @@
+name: gql
+description: ⚡️ Minimal GraphQL Client + Code Generation for Nuxt3. Auto imports GraphQL Operations for easy execution.
+repo: diizzayy/nuxt-gql
+npm: nuxt-gql
+icon: gql.png
```

Comment:
The added logo seems from an article with Apollo integration. Maybe directly using svg from graphql-code-generator? 

https://www.graphql-code-generator.com/assets/subheader-logo.svg